core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -2207,14 +2207,27 @@ private[spark] object Utils extends Logging {
   }
 
   /**
+   * Return whether it is local mode
+   */
+  def isLocal(conf: SparkConf): Boolean = {
```

Comment:
Where did you feel unclear ? I think it is straightforward. And I think @return is only mandatory when it is public api.